encrypted-media-respec.html

```diff
@@ -2301,9 +2303,92 @@
             <li><p>Resolve <var>promise</var>.</p></li>
           </ol>
         </section>
-      </section>
 
-      <section>
+        <section id="monitor-cdm">
+          <h4>Monitor for CDM Changes</h4>
```

Comment:
Should we use "State" in the name? Otherwise, the name could imply just that the CDM changed (i.e. was switched out). "Session State" seems better, but that does not cover the last step. Still, it might be better to be more descriptive and just have that as an extra thing it does.